config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
is it intentional to drop `:password` from this list?  Not sure if this is rails defaults overriding our values.

Reply:
filter parameters are substring match, so passw includes password.  This is the same original file, so we didn't "add" password originally as it was the default.